v3/setup/capcom.sh

```diff
@@ -7,26 +7,24 @@ if [ "${NODE_ROLE}" != "control" ]  && [ "${NODE_ROLE}" != "it-hybrid" ]; then
     exit 0
 fi
 
-if [ "$(etcdctl get capcom-bootstrapped)" == "true" ]; then
-    exit 0
+etcdctl get /capcom/config/proxy
```

Comment:
I don't love this because if I have to rerun the config load, I currently set the `bootstrapped` flag to false and then run it. This will mean I have to delete all the keys.